.github/workflows/ci.yml

```diff
@@ -86,4 +86,24 @@ jobs:
         with:
           go-version-file: go.mod
           cache: false
+
+      - uses: elastic/apm-pipeline-library/.github/actions/docker-login@current
+        with:
+          registry: docker.elastic.co
+          secret: secret/observability-team/ci/docker-registry/prod
+          url: ${{ secrets.VAULT_ADDR }}
+          roleId: ${{ secrets.VAULT_ROLE_ID }}
+          secretId: ${{ secrets.VAULT_SECRET_ID }}
+        if: |
```

Comment:
Only when working on branches in the upstream or PRs coming from the upstream then login, GitHub secrets are not available on forked PRs.